lib/matplotlib/backend_tools.py

```diff
@@ -853,7 +852,7 @@ def _cancel_action(self):
         return
 
     def _press(self, event):
-        """the _press mouse button in zoom to rect mode callback"""
+        """Callback for mouse button presses in zoom to rect mode."""
```

Comment:
```suggestion
        """Callback for mouse button presses in zoom-to-rect mode."""
```
I would use hyphens for better readability, but not a native speaker, so not sure that's correct.

(occurs multiple times also in docstrings below)

Reply:
done

Reply:
I don't see this?

Reply:
Hum, sorry, forgot to push.  Done.